components/_util/wave/style.ts

```diff
@@ -32,8 +32,8 @@ const genWaveStyle: GenerateStyle<WaveToken> = (token) => {
 
         '&.wave-quick': {
           transition: [
-            `box-shadow 0.3s ${token.motionEaseInOut}`,
-            `opacity 0.35s ${token.motionEaseInOut}`,
+            `box-shadow ${token.motion ? 0.3 : 0}s ${token.motionEaseInOut}`,
+            `opacity ${token.motion ? 0.35 : 0}s ${token.motionEaseInOut}`,
```

Comment:
It is better to disable motion in https://github.com/ant-design/ant-design/blob/master/components/_util/wave/index.ts

Reply:
ping

Reply:
ping